package.json

```diff
@@ -10,8 +10,10 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "daisyui": "^4.7.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-icons": "^5.0.1",
```

Comment:
Is there any need of this package as you can directly download icons form figma file?